src/NServiceBus.AcceptanceTests/DelayedDelivery/When_using_special_characters_in_headers.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_using_special_characters_in_headers : NServiceBusAcceptanceTest
+    {
+        readonly Dictionary<string, string> specialHeaders = new Dictionary<string, string>
+        {
+            { "a-B1", "a-B" },
+            { "a-B2", "a-ɤϡ֎ᾣ♥-b" },
+            { "a-ɤϡ֎ᾣ♥-B3", "a-B" },
+            { "a-B4", "a-\U0001F60D-b" },
+            { "a-\U0001F605-B5", "a-B" },
+            { "a-B6", "a-😍-b" },
+            { "a-😅-B7", "a-B" },
+        };
+
+        [Test]
+        public async Task Should_store_unicode_characters_in_timeout_persistence_for_delayed_messages()
+        {
+            Requires.TimeoutStorage();
+
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<EndpointHandlingDelayedMessages>(e => e
+                    .When(s =>
+                    {
+                        var options = new SendOptions();
+                        options.RouteToThisEndpoint();
+                        options.DelayDeliveryWith(TimeSpan.FromSeconds(3));
```

Comment:
yes with that value it should go through the timeout storage (see https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/DelayedDelivery/TimeoutManager/StoreTimeoutBehavior.cs#L64)